test.sh

```diff
@@ -29,4 +29,5 @@ go vet $(go list ./... | grep -v vendor)
 
 remove_containers
 
+curl -H 'Accept: application/json' -X GET https://mermade.org.uk/api/v1/validate?url=https://gist.githubusercontent.com/denismakogon/f1c306d771480bbe39b363de070e432f/raw/ad8ef852e88c0cd8fcfbe52d7cfaa6d85ee62d1f/gistfile1.txt | go run test/openapi-validate.go
```

Comment:
That gist contains the same spec as this PR.

P.S.: i nearly crashed my brain to develop go tool to validate spec with API endpoints of the online validator, it just doesn't work with cURL (if i attempt to validate with the file as the payload, it works only in the browser).